src/main/thrift/parquet.thrift

```diff
@@ -712,9 +712,16 @@ struct DictionaryPageHeader {
 }
 
 /**
- * New page format allowing reading levels without decompressing the data
+ * Alternate page format allowing reading levels without decompressing the data
  * Repetition and definition levels are uncompressed
  * The remaining section containing the data is compressed if is_compressed is true
+ *
+ * Implementation note - this header is not necessarily a strict improvement over
+ * `DataPageHeader` (in particular the original header might provide better compression 
+ * in some scenarios). Page indexes require pages start and end at row boundaries regardless of which
+ * page header is used.
+ * https://parquet.apache.org/docs/file-format/implementationstatus/ tracks the implementation of readers and 
+ * writers that support this page format. 
```

Comment:
It does not seem useful to provide this link here as it would apply to most other Parquet features as well.